projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3873,52 +3881,51 @@ public function deactivate_dialog( $hook ) {
 			&& self::is_connection_ready()
 		) {
 
-			$active_plugins_using_connection = Connection_Plugin_Storage::get_all();
+			// Register jp-tracks-functions dependency.
+			Tracking::register_tracks_functions_scripts( true );
 
-			if ( count( $active_plugins_using_connection ) > 1 ) {
-
-				add_thickbox();
-
-				// Register jp-tracks-functions dependency.
-				Tracking::register_tracks_functions_scripts();
-
-				wp_enqueue_script(
-					'jetpack-deactivate-dialog-js',
-					Assets::get_file_url_for_environment(
-						'_inc/build/jetpack-deactivate-dialog.min.js',
-						'_inc/jetpack-deactivate-dialog.js'
+			// add a deactivation script that will pick up deactivation actions for the Jetpack plugin.
+			Assets::register_script(
+				'jetpack-plugins-page-js',
+				'_inc/build/plugins-page.js',
+				JETPACK__PLUGIN_FILE,
+				array(
+					'in_footer'    => true,
+					'dependencies' => array(
+						'wp-polyfill',
+						'wp-components',
```

Comment:
I would recommend adding the textdomain, so translations can be handled.

```suggestion
					'textdomain'   => 'jetpack'
					'dependencies' => array(
						'wp-polyfill',
						'wp-components',
```